en/V4-Communication_Requirements.md

```diff
@@ -2,15 +2,13 @@
 
 ## Control Objective
 
-Ensure that a verified application satisfies the following high level requirements:
+Devices use network communication to exchange data and receive commands with its ecosystem. So that the different parties can trust the contents of communications, they need to be protected, ensuring authenticity of the parties, integrity against malicious changes, and confidentiality against information leakage. In practice, this translates to deploying up-to-date communication protocols and configuring their security features, including cryptography. Since industry guidelines on secure TLS, Bluetooth or Wi-Fi  configuration changes frequently, configuration should be periodically reviewed to ensure that communications security is always effective.

```

Comment:
Proposed edits below:

Devices use network communication to exchange data and receive commands within their ecosystem. So that the different parties can trust the contents of communications, they need to be protected, ensuring the authenticity of parties, integrity against malicious changes, and confidentiality against information leakage. In practice, this translates to deploying up-to-date communication protocols and configuring their security features, including cryptography. Since industry guidelines on secure TLS, Bluetooth, and Wi-Fi change frequently, configurations should be periodically reviewed to ensure that communications security is always effective.